### DISCUSSION TITLE
Does Deepgram delete user data after transcription?

### DISCUSSION THREAD

**Discussion by feiwualice** [deepgram/community#115] [General help] [answered]
I would like to delete my own dataset after using Deepgram to process the data. But I didn't find any information about how to delete it in Deepgram website.

Anyone know if Deepgram store our data sets and is there any ways for us to delete dataset after processing it?

**Comment by briancbarrow** [comment]
@feiwualice Let me double check on this. When you say "dataset" do you mean the audio data, or the transcription data?

**Comment by andiaziz** [comment]
Hi deepgram, do you store the audio file when I request the transcription?

**Comment by lukeocodes** [comment, marked as answer]
This is answered pretty concisely by our [Terms](https://deepgram.com/terms/) and our [Privacy Policy](https://deepgram.com/privacy/). Please see those pages for all information on user data retention policy.

**Comment by altaha** [comment]
@lukeocodes Thank you for providing the links. But as someone who is not qualified to dive in and understand the legalese, I would appreciate an answer to the following to help me select a vendor for a startup:

1. Does deepgram store the audio for a long term period (beyond a few days) - regardless of whether it is anonymized, encrypted etc? Does any submitted audio get retained by Deepgram for model-training or other purposes? If so, is there is a simple way to opt-out?

2. Does deepgram store the transcribed text for a long term period (beyond a few days) - regardless of whether it is anonymized, encrypted etc? Does any transcribed text get retained by Deepgram for model-training or other purposes? If so, is there is a simple way to opt-out?

Thanks!

**Reply by lukeocodes** [reply]
@altaha as mentioned previously, this is answered in our terms and our privacy notice, and is not deliberately obscure.

Please see part 3 from our [terms](https://deepgram.com/terms/):

And as detailed in our [privacy notice](https://deepgram.com/privacy/), we adhere to all requirements for data removal under governmental regulations. i.e. GDPR, CCPA.

If you have a specific request for data removal, or more questions about this, please direct your request to [security@deepgram.com](mailto:security@deepgram.com).
